cmd/release/config/config.go

```diff
@@ -102,89 +88,49 @@ type Config struct {
 	Auth    *Auth          `json:"auth"`
 }
 
-func DefaultConfigPath() (string, error) {
-	homeDir, err := os.UserHomeDir()
-	if err != nil {
-		return "", nil
-	}
-
-	return filepath.Join(homeDir, ecmDistroDir, configFileName), nil
-}
-
 // Load reads the given config file and returns a struct
 // containing the necessary values to perform a release.
-func Load(configFile string) (*Config, error) {
-	f, err := os.Open(configFile)
-	if err != nil {
-		return nil, err
-	}
-
-	return read(f)
-}
-
-func read(r io.Reader) (*Config, error) {
-	var c Config
-	if err := json.NewDecoder(r).Decode(&c); err != nil {
-		return nil, err
-	}
-
-	return &c, nil
-}
-
-func OpenOnEditor() error {
-	confPath, err := DefaultConfigPath()
-	if err != nil {
-		return err
-	}
-
-	cmd := exec.Command(textEditorName(), confPath)
+func OpenOnEditor(configFile string) error {
+	cmd := exec.Command(textEditorName(), configFile)
 	cmd.Stdin = os.Stdin
 	cmd.Stdout = os.Stdout
 
 	return cmd.Run()
 }
 
-func Generate() error {
-	configExists := true
-
-	configPath, err := DefaultConfigPath()
-	if err != nil {
-		return err
-	}
-
-	if _, err := os.Stat(configPath); err != nil {
-		if !strings.Contains(err.Error(), "no such file or directory") {
-			return err
-		}
-		configExists = false
-	}
-	if configExists {
-		return errors.New("config already exists at " + configPath)
+func textEditorName() string {
+	editor := os.Getenv("EDITOR")
+	if editor == "" {
+		editor = "vi"
 	}
 
-	confB, err := json.MarshalIndent(exampleConfig(), "", " ")
+	return editor
+}
+func Load(configFile string) (*Config, error) {
```

Comment:
Is this function or the one below used outside of this package?

Reply:
Yes, they are both used by the `cmd` package